core/create_spoken_forms.py

```diff
@@ -14,11 +14,8 @@
 mod = Module()
 
 
-# TODO: 'Whats application': 'WhatsApp' (Should keep "whats app" as well?)
-# TODO: 'V O X': 'VOX' (should keep "VOX" as well?)
-# Could handle by handling all alternatives for these, or by having hardcoded list of things that we want to handle specially
-# TODO: Tests
-DEFAULT_MINIMUM_TERM_LENGTH = 3
+DEFAULT_MINIMUM_TERM_LENGTH = 2
```

Comment:
just curious, why did you drop this to 2? I recall settling on three for _*waves hands*_ reasons lost to time.

Reply:
I believe I dropped it to 2 because otherwise if you have a file like say vm.md or a folder vm/ a value of 3 wouldn't allow you to access it by saying "V M". I'll double check though when I implement three tests.

Reply:
That makes sense, thanks